nodejs/week2/homework/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -0,0 +1,65 @@
+const express = require("express");
+const router = express.Router();
+
+const meals = require("./../data/meals.json");
+
+router.get("/", async (request, response) => {
+  try {
```

Comment:
This is optional and only if you want practice further!
What if I want to use more than one query parameter, for instance have both title and maxPrice? Currently you are using response.send directly, so I will get results based on only the first parameter. What could be done differently?